addons/test_mail/models/test_mail_corner_case_models.py

```diff
@@ -65,6 +65,15 @@ class MailTestTrackCompute(models.Model):
     partner_phone = fields.Char(related='partner_id.phone', tracking=True)
 
 
+class MailTestTrackBinary(models.Model):
```

Comment:
Would just define it before "mail.test.track.compute" (arbitrary alphabetical ordering :) )